.github/workflows/build-docker.yml

```diff
@@ -115,5 +118,22 @@ jobs:
             export LDFLAGS=-Wl,--no-as-needed ${LDFLAGS}
           fi
           ./scripts/ci/test-examples.sh
+      - name: Upload test results
+        # Note: upload-v3 doesn't seem to pick up the paths: delete the "false" line below once we 
+        # upgrade the image
+        if: >-
+          ${{
+            false &&
+            always()
+            && (steps.test.outcome == 'success'
+                || steps.test.outcome == 'failure')
+            && (env.CMAKE_PRESET != 'ndebug-vecgeom')
+          }}
+        uses: actions/upload-artifact@v3
```

Comment:
Why `v3` here and `v4` later?

Reply:
The docker image is stuck on an old version of centos/ubuntu that doesn't support the newer version of node.js ... we're overdue for regenerating these images but I just haven't made it a priority since I've been able to patch them along.